nautobot/circuits/models.py

```diff
@@ -240,7 +307,21 @@ class Meta:
         unique_together = ["circuit", "term_side"]
 
     def __str__(self):
-        return "Side {}".format(self.get_term_side_display())
+        return f"Termination {self.term_side}: {self.site or self.provider_network}"
+
+    def get_absolute_url(self):
+        if self.site:
+            return self.site.get_absolute_url()
+        return self.provider_network.get_absolute_url()
```

Comment:
I'm not clear on why we'd want the `absolute_url` to be for the `site` if it's set. I see in other code context how it's conditional, however, can you please add a comment of the intent here for posterity?